inference-engine/src/readers/ir_reader/CMakeLists.txt

```diff
@@ -29,9 +29,11 @@ target_compile_definitions(${TARGET_NAME} PRIVATE IMPLEMENT_INFERENCE_ENGINE_API
 
 target_include_directories(${TARGET_NAME} PUBLIC ${PUBLIC_HEADERS_DIR})
 target_include_directories(${TARGET_NAME} PRIVATE "${IE_MAIN_SOURCE_DIR}/src/inference_engine")
+target_include_directories(${TARGET_NAME} PRIVATE "${COMMON_INCLUDE_PATH}")
 
 target_link_libraries(${TARGET_NAME} PUBLIC inference_engine_reader_api inference_engine_plugin_api ${NGRAPH_LIBRARIES} inference_engine)
 target_link_libraries(${TARGET_NAME} PRIVATE pugixml)
+target_link_libraries(${TARGET_NAME} PRIVATE common)
```

Comment:
we just need to add `common` library as an interface library for plugin API. So, all these Cmake changes will not be needed:
```cmake
target_link_libraries(inference_engine_plugin_api INTERFACE common)
```